app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -58,7 +59,9 @@ const MakeDecision = ({ data }) => {
     },
     decisionType: SELECT_DEFECT_MANUALLY,
     issueActionType: '',
-    optionValue: CURRENT_EXECUTION_ONLY,
+    optionValue: itemData.issue.issueType.startsWith('ti')
+      ? SIMILAR_TI_CURRENT_LAUNCH
+      : CURRENT_EXECUTION_ONLY,
     searchMode: '',
```

Comment:
`searchMode` depends on an option which you can choose on second step, please provide corresponding `searchMode` in case by default was chosen `similar TI...` You can see it in `OptionsBlock` component `onChangeOption` func.